docs/cli/edgedb_branch/edgedb_branch_rebase.rst

```diff
@@ -26,6 +26,9 @@ Description
 Creates a new branch that is based on the target branch, but also contains any new migrations on the
```

Comment:
maybe we should provide here a more info about how rebasing works, sth like: Creates a new branch that is based on the target branch, and also contains new migrations from the current branch. Drops the current branch and rename new branch to the current branch name.  (this is not the best but I needed a bit more info when working with rebasing)